controllers/medusa/medusaconfiguration_controller_test.go

```diff
@@ -39,8 +39,8 @@ func testMedusaConfigurationOk(t *testing.T, ctx context.Context, f *framework.F
 	}
 
 	t.Log("Creating medusa bucket key secret")
-	err := f.Client.Create(ctx, bucketKeySecret)
-	require.NoError(err, "failed to create medusa bucket key secret")
+	// err := f.Client.Create(ctx, bucketKeySecret)
```

Comment:
Here, and few lines below, perhaps we should remove the commented lines in order to not confuse a future reader.

Reply:
Ye.